Makefile

```diff
@@ -1,22 +1,20 @@
 PKG := github.com/btcsuite/btcd
 
 LINT_PKG := github.com/golangci/golangci-lint/cmd/golangci-lint
-GOACC_PKG := github.com/ory/go-acc
 GOIMPORTS_PKG := golang.org/x/tools/cmd/goimports
 
 GO_BIN := ${GOPATH}/bin
 LINT_BIN := $(GO_BIN)/golangci-lint
-GOACC_BIN := $(GO_BIN)/go-acc
 
 LINT_COMMIT := v1.18.0
-GOACC_COMMIT := 80342ae2e0fcf265e99e76bcc4efd022c7c3811b
 
 DEPGET := cd /tmp && go install -v
 GOBUILD := go build -v
 GOINSTALL := go install -v 
 DEV_TAGS := rpctest
 GOTEST_DEV = go test -v -tags=$(DEV_TAGS)
 GOTEST := go test -v
+COVER_FLAGS = -coverprofile=coverage.txt -covermode=atomic
```

Comment:
I think we also need `-coverpkg=$(PKG)/...` otherwise we might get incorrect results if one package indirectly covers another package in the tests.